plugins/nodemon/src/tasks/nodemon.ts

```diff
@@ -36,7 +36,12 @@ export default class Nodemon extends Task<typeof NodemonSchema> {
       PORT: port.toString(),
       ...process.env
     }
-    nodemon({ script: entry, env, stdout: false, configFile: configPath })
+    const config: nodemon.Settings = { script: entry, env, stdout: false, configFile: configPath }
+    // disable native fetch if supported by runtime
+    if (process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch')) {
+      config.execMap = { js: 'node --no-experimental-fetch' }
```

Comment:
**suggestion**: in `node` we're forwarding the parent's `execArgv`, do we want to do that here as well?

Reply:
it's not the default behaviour of `nodemon` to forward the `execArgv` so idk I think that might be surprising?